deploy/operator/deploy.sh

```diff
@@ -3,24 +3,64 @@
 set -o nounset
 set -o pipefail
 set -o errexit
-set -o xtrace
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(realpath ${__dir}/../..)"
 
 if [ -z "${DISKS:-}" ]; then
     export DISKS=$(echo sd{b..f})
 fi
 
+function handle_disconnected() {
+    # Some of the variables over here can be sourced from dev-scripts
+    # source common.sh utils.sh
+    # set +x
+    # export -f wrap_if_ipv6 ipversion
+
+    source ${__dir}/mirror_utils.sh
+    export MIRROR_BASE_URL="http://$(wrap_if_ipv6 ${PROVISIONING_HOST_IP})/images"
+    export AUTHFILE="${XDG_RUNTIME_DIR}/containers/auth.json"
+    mkdir -p $(dirname ${AUTHFILE})
+
+    export LOCAL_REGISTRY="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}"
```

Comment:
hmm, I don't 100% agree with the choice of external variables here
imho ``MIRROR_BASE_URL``, ``AUTHFILE`` and ``LOCAL_REGISTRY`` are better options to be given by the user than ``PROVISIONING_HOST_IP``, ``XDG_RUNTIME_DIR`` and ``LOCAL_REGISTRY_*``. This will give a better abstraction to work with

Reply:
This is the script copied from `openshift/release` to `assisted-service` repo.
The idea is to make a user be able just to deploy without thinking. If you want to override anything we can do that, but I don't think that's a necessary ATM since no one would do that...
What do you suggest?

Reply:
So let's compromise and enable providing ``MIRROR_BASE_URL``, ``AUTHFILE`` etc. which will be taken first-priority
E.g.:
```
export MIRROR_BASE_URL="${MIRROR_BASE_URL:-http://$(wrap_if_ipv6 ${PROVISIONING_HOST_IP})/images}"
export LOCAL_REGISTRY="${LOCAL_REGISTRY:-${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}}"
```
And we can make some changes on dev-scripts as well, to make it easier to use
What do you think?

Reply:
Anyway, not a blocker
But I think that's what we should strive to